doc/release/README.compiling

```diff
@@ -37,8 +37,7 @@ We note some of the most useful flags here:
 
      -O : turns on optimization of the generated C code
 
-     --fast : turns on --no-checks, -O, and relaxes IEEE floating
-              point strictness
+     --fast : turns on --no-checks, -O, and enables many other optimizations
```

Comment:
It looks like fNoVectorize is by default true (so --fast enables vectorization). I thought most of these were off by default... but they aren't.